docs/docs/how-to/cloud/enable-image-cdn.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
This doc is still not in a state I'd be comfortable merging. It's missing this block from my previous comment:

> But this block of information needs to be rephrased and moved completely anyways, this needs to be in the "Prerequisites" step. It would say:
> You have to have these packages installed (if not done yet, here's how you install them).
> At the moment the document talks about "upgrading" dependencies, that's wrong as the How-To guides always assume things with the Prerequisites.

And also the information from the reference guide that should be moved here. And my other comments in this doc here.

Rewriting this doc from scratch in accordance to the "How-To" guidelines will really help, because right now it's a bad migration guide, not a how-to guide that people can actually follow.

Reply:
@smthomas - As a heads up, it seems there are multiple concerns from @LekoArts on the form and function of this page (which we initially simply migrated from the current cloud knowledge base and formatted).

I'm going to remove this page from the initial set of migrated pages, and we can deal with this one separately. 

Thanks for maintaining the quality bar @LekoArts

Reply:
I've removed the page, and also removed from the docs sidebar until the document is more deeply reviewed by Shane and team.